include/rfb.js

```diff
@@ -432,7 +432,21 @@ var RFB;
          *   ClientInitialization - not triggered by server message
          *   ServerInitialization (to normal)
          */
+        _stateQueue: new Array(),
```

Comment:
yeah, otherwise, if you have two instances of the RFB object, they'll be using the same queue.